packages/email-petition/src/email-petition.tsx

```diff
@@ -35,16 +34,29 @@ import { Alert, AlertDescription, AlertTitle } from "@dxe/petitions-components/a
 import { LoaderIcon, MailCheckIcon } from "lucide-react";
 import ReCAPTCHA from "react-google-recaptcha";
 
-const PETITION_API_URL = "https://petitions-229503.appspot.com/api/sign";
+const PETITION_API_URL = `${process.env.NEXT_PUBLIC_PETITIONS_API_ROOT}/sign`;
 
 const CAMPAIGN_MAILER_API_URL = `${process.env.NEXT_PUBLIC_CAMPAIGN_MAILER_API_ROOT}/message/create`;
-const CAMPAIGN_NAME = process.env.NEXT_PUBLIC_CAMPAIGN_NAME;
-console.log("mailer url: " + CAMPAIGN_MAILER_API_URL);
-console.log("campaign: " + CAMPAIGN_NAME);
 
 const CAPTCHA_SITE_KEY = "6LdiglcpAAAAAM9XE_TNnAiZ22NR9nSRxHMOFn8E";
 
-export const Petition = () => {
+export function EmailPetition(
+  props: {
+    petitionId: string,
+    campaignName: string,
+    defaultMessage: string,
+    onSubmit?: () => void
+  }) {
+
+  useEffect(() => {
```

Comment:
purposefully leaving this?

Reply:
right oops about that, what would you think about me accepting a ?debug=true query param and pass it to the <EmailPetition> component and conditionally log this?

i removed it for now.

(btw for testing production setup, npm start doesn't work with exported sites but it told me to do "pnpx serve@latest out" and that worked.)